shell/platform/darwin/ios/ios_surface_metal_impeller.mm

```diff
@@ -64,7 +66,7 @@
   // exit the app.
   layer.presentsWithTransaction = [[NSThread currentThread] isMainThread];
 
-  return layer;
+  return (__bridge GPUCAMetalLayerHandle)layer;
```

Comment:
I think you'll need `__attribute((ns_returns_not_retained))` in the header to match the previous semantics and the scoped_nsobject::get semantics.